ee/control/client_http.go

```diff
@@ -150,6 +150,42 @@ func (c *HTTPClient) GetSubsystemData(hash string) (io.Reader, error) {
 	return reader, nil
 }
 
+// MessageServer sends a message to the server using JSON-RPC format
+func (c *HTTPClient) MessageServer(method string, params interface{}) error {
```

Comment:
I'd name this `MessageSender` "server" sounds like it's the receiver portion

Reply:
How about just `Message`? `MessageSender` feels strange to me because this in http client and the client does the "sending".